lib/src/clojure_lsp/kondo.clj

```diff
@@ -114,10 +114,10 @@
        name-end-row
        name-end-col))
 
-(defn normalize-analysis [analysis]
+(defn normalize-analysis [external? analysis]
```

Comment:
Could accept a hashmap `{:keys [external?]}`, if you think we'll add more context over time. Might not be necessary though.

Reply:
Yeah, maybe we can follow 1..2..n rule here, and change it next time we need a third one